### ISSUE TITLE
Appstore Versions Out Of Sync

### ISSUE THREAD

**Issue by mkumpf** [ako/CsvServices#4]
Andrej,

I have just tried to download CSV services version 2.0.1 and 2.0 into an app that is in Mendix version 6.10.9 and I get the following error message:

![image](https://user-images.githubusercontent.com/3680543/31646489-13dcf602-b2d0-11e7-983e-e1c7f939d9f6.png)

Is there an up to date version that is available for Mendix 6.10.x?
Thanks,
Mike

**Comment by ako** [comment]
No, i'm not maintaining versions for older mendix releases. Release 2.3 is Mendix 7.23.8 and up.
